packages/client/src/modules/user/containers/Users.native.jsx

```diff
@@ -2,49 +2,28 @@ import React from 'react';
 import { StyleSheet, View } from 'react-native';
 import PropTypes from 'prop-types';
 import { compose } from 'react-apollo';
+
 import UsersList from '../components/UsersListView';
 import UsersFilter from '../components/UsersFilterView';
+import withSubscription from './UsersWithSubscription';
```

Comment:
File name must match the name of default import. In this case the file should named `withSubscription`